src/FlipCounter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      x: 0
+  
+    };
+  }
+  add = (num) => {
+    this.setState({ x: parseInt(this.state.x) + 1 });
+    console.log("add activated");
+    console.log(this.state.x);
+  };
+ 
+ 
+ 
+  myChangeHandler = (event) => {
+    this.setState({ value: event.target.value });
+    console.log("change activated");
+  };
+
+  mySubmitHandler = (event) => {
+    this.setState({ x: this.state.value });
+    console.log("submit activated");
+    event.preventDefault();
+  };
+
+  render() {
+   console.log(this.state.x)
+   console.log(this.state.value)
+   const arr = [
```

Comment:
A really verboose way to split the digit to separate numbers.

This about this in other way: you do not need a number to render it in JSX. Even more: you render something as string.

How to make it easier: convert number to string, add leading zeroes and create an array of characters out of a string.